### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by u6684258**
Three main changes:
1. update the tokeniser to the latest version to support necessary PDDL keywords;
2. update the fast-downward API as some function names have changed in the latest version (planner methodology not changed)
3. update the singularity file for bug fixing

### CODE REVIEW

File: Apptainer.asnets.plan [hunk 1]
```diff
@@ -19,7 +18,7 @@ Stage: build
         libhdf5-dev g++ make git 
 
     pip3 install --upgrade pip
-    pip3 install wheel cython numpy pkgconfig werkzeug
+    pip3 install wheel cython numpy pkgconfig werkzeug urllib3==1.26.15
```
Comment (jendrikseipp) [inline Apptainer.asnets.plan:21]: Can you please pin the versions of the other Python packages on this line? This ensures that the code works in the future as well.

File: Apptainer.asnets.learn [hunk 2]
```diff
@@ -19,7 +18,7 @@ Stage: build
         libhdf5-dev g++ make git 
 
     pip3 install --upgrade pip
-    pip3 install wheel cython numpy pkgconfig werkzeug
+    pip3 install wheel cython numpy pkgconfig werkzeug urllib3==1.26.15
```
Comment (jendrikseipp) [inline Apptainer.asnets.learn]: Please specify the versions of the other packages on this line as well.

File: asnets/asnets/interfaces/fd_interface.py [hunk 3]
```diff
@@ -194,7 +194,7 @@ def make_wlama_args(w):
         # fast-downward.py --alias lama-first ${dom} ${prob}
         cmdline.extend([
             domain_bn, problem_bn, "--evaluator",
-            ("hlm=lmcount(lm_factory=lm_rhw(reasonable_orders=true),"
+            ("hlm=landmark_sum(lm_factory=lm_rhw(),"
```
Comment (jendrikseipp) [inline asnets/asnets/interfaces/fd_interface.py]:
```suggestion
            ("hlm=landmark_sum(lm_factory=lm_reasonable_orders_hps(lm_rhw()),"
```
This preserves the reasonable_orders=true flag. Please check that the planner call still works.

**Review by jendrikseipp — CHANGES_REQUESTED** [review]

File: asnets/asnets/interfaces/fd_interface.py (hunk 3 above)
Comment (u6684258) [reply asnets/asnets/interfaces/fd_interface.py]: Thanks for the suggestion : )

**Pull request merged**